packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -29,6 +29,10 @@ $Rating-largeIconSize: 20px;
         & ~ .star {
           // Set default color for any star after hovered one
           color: $Rating-defaultColor;
+
+          .halfStar{
```

Comment:
why do we need special styling for that? It should be the same color as `.star`

+formatting